format/Layout.md

```diff
@@ -90,11 +90,27 @@ maximum of 2^31 - 1 elements. We choose a signed int32 for a couple reasons:
 Any relative type can be nullable or non-nullable.
 
 Nullable arrays have a contiguous memory buffer, known as the null bitmask,
```

Comment:
It might be better to frame this paragraph in terms of validity.  Drills documentation phrases it well I think (it doesn't reference a null bitmark)

> Nullable values are represented by a vector of bit values. Each bit in the vector corresponds to an element in the ValueVector. If the bit is not set, the value is NULL.